Rakefile

```diff
@@ -10,6 +10,13 @@ RSpec::Core::RakeTask.new(:unit) do |t,args|
   t.verbose     = true
 end
 
+desc "Integration tests"
```

Comment:
I ported the tests from the DSC module which were placed in the integration directory.  There was no existing rake task for it, so I created one.